lib/generators/active_storage/blurhash/install/install_generator.rb

```diff
@@ -15,7 +15,13 @@ def copy_application_javascript
     directory "javascript", "app/javascript/blurhash"
   end
 
+  def pin_blurhash_javascript
+    if File.exist? Rails.root.join("config", "importmap.rb")
+      append_to_file "config/importmap.rb", 'pin "active_storage_blurhash", to: "blurhash/index.js"' + "\n"
+    end
+  end
+
   def append_to_main_javascript_entrypoint
-    append_to_file "app/javascript/application.js", "import \"./blurhash\";\n"
+    append_to_file "app/javascript/application.js", "import \"active_storage_blurhash\";\n"
```

Comment:
I think we need to be more careful here, because this would break non-importmap applications who have no knowledge of an `active_storage_blurhash` package.

Maybe add a switch like this? 
https://github.com/avo-hq/active_storage-blurhash/pull/8/files#diff-6e19a81d79f6d19aa21abc1fb51d1cb233afbdae097f2b9758a77dbaafc12408L5

Note that @adrianthedev and me opted against making this a Javascript package of its own right, though we might want to reconsider this at some point 🤔